monailabel/scripts/run_monailabel_user_app.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
> since we use direct file to run the command (instead of sh <filename>), when you git commit, please make sure you have +x permissions for this file.